tests/attributes.py

```diff
@@ -1,13 +1,17 @@
+from django.core.exceptions import ValidationError
 from django.test import TestCase
 
 import eav
-from eav.registry import EavConfig
 from eav.models import Attribute, Value
+from eav.registry import EavConfig
 
-from .models import Patient, Encounter
+from .models import Encounter, Patient
 
 
-class LimittingAttributes(TestCase):
+class Attributes(TestCase):
+    '''
+    TODO: Explain this test.
```

Comment:
Ah, the mythic land of Later :D

Reply:
Oh, yeah :smile: 

The test itself is old, I just added the note for myself (to document it later on the main branch, since this is a bit of out of the scope of this pull request).

Reply:
OK